colorbleed/maya/lib.py

```diff
@@ -837,6 +847,7 @@ def set_id(node, unique_id, overwrite=False):
 
     # Set the value
     if not exists or overwrite:
+        attr = "{0}.cbId".format(node)
```

Comment:
If we go for performance we could go for the old skool formatting approach, `%s.cbId % node` . This is also micro improvements ;)